src/controllers/queryRunner.ts

```diff
@@ -307,7 +307,6 @@ export default class QueryRunner {
 			const batchSet = this.batchSets[batchId];
 			this.eventEmitter.emit('batchStart', batchSet);
 			let executionTime = <number>(Utils.parseTimeString(batchSet.executionElapsed) || 0);
-			this._totalElapsedMilliseconds += executionTime;
```

Comment:
When this line is present, execution time continues to change on tab switch. Here's a gif of this happening:

![Total Execution Time - Change](https://github.com/microsoft/vscode-mssql/assets/87730006/21d158fc-bb44-4584-bd12-8fcd38d4d600)



When this line is not present, execution time doesn't change as seen in this other gif:

![Total Execution Time - No change](https://github.com/microsoft/vscode-mssql/assets/87730006/5cfb8bdb-6a9a-44c2-9be4-3f79bf126422)